subsys/bluetooth/services/ots/ots_oacp.c

```diff
@@ -23,7 +25,7 @@
 LOG_MODULE_DECLARE(bt_ots, CONFIG_BT_OTS_LOG_LEVEL);
 
 #define OACP_PROC_TYPE_SIZE	1
-#define OACP_RES_MAX_SIZE	3
+#define OACP_RES_MAX_SIZE	(3 + sizeof(uint32_t))
```

Comment:
Please add a comment stating why it's 3 + 4 (i.e. what fields are making this MAX size)

Reply:
done

Reply:
Need more details: Why is it 3?

Reply:
Add complete comment of OACP response.